src/parcsr_ls/F90_HYPRE_parcsr_mgr.c

```diff
@@ -0,0 +1,807 @@
+                hypre_F90_PassObj (HYPRE_ParCSRMatrix, A),
+                hypre_F90_PassObj (HYPRE_ParVector, b),
+                hypre_F90_PassObj (HYPRE_ParVector, x) ) );
+}
+
+/*--------------------------------------------------------------------------
+ * HYPRE_MGRSolve
+ *--------------------------------------------------------------------------*/
+
+void
+hypre_F90_IFACE(hypre_mgrsolve, HYPRE_MGRSOLVE)
+( hypre_F90_Obj *solver,
+  hypre_F90_Obj *A,
+  hypre_F90_Obj *b,
+  hypre_F90_Obj *x,
+  hypre_F90_Int *ierr )
+{
+   *ierr = (hypre_F90_Int)
+           ( HYPRE_MGRSolve(
+                hypre_F90_PassObj (HYPRE_Solver, solver),
+                hypre_F90_PassObj (HYPRE_ParCSRMatrix, A),
+                hypre_F90_PassObj (HYPRE_ParVector, b),
+                hypre_F90_PassObj (HYPRE_ParVector, x) ) );
+}
+
+#ifdef HYPRE_USING_DSUPERLU
+
+/*--------------------------------------------------------------------------
+ * HYPRE_MGRDirectSolverCreate
+ *--------------------------------------------------------------------------*/
+
+void
+hypre_F90_IFACE(hypre_mgrdirectsolvercreate, HYPRE_MGRDIRECTSOLVERCREATE)
+( hypre_F90_Obj *solver,
+  hypre_F90_Int *ierr )
+{
+   *ierr = (hypre_F90_Int)
+           ( HYPRE_MGRDirectSolverCreate(
+                hypre_F90_PassObjRef (HYPRE_Solver, solver) ) );
+}
+
+/*--------------------------------------------------------------------------
+ * HYPRE_MGRDirectSolverDestroy
+ *--------------------------------------------------------------------------*/
+
+void
+hypre_F90_IFACE(hypre_mgrdirectsolverdestroy, HYPRE_MGRDIRECTSOLVERDESTROY)
+( hypre_F90_Obj *solver,
+  hypre_F90_Int *ierr )
+{
+   *ierr = (hypre_F90_Int)
+           ( HYPRE_MGRDirectSolverDestroy(
+                hypre_F90_PassObj (HYPRE_Solver, solver) ) );
+}
+
+/*--------------------------------------------------------------------------
+ * HYPRE_MGRDirectSolverSetup
+ *--------------------------------------------------------------------------*/
+
+void
+hypre_F90_IFACE(hypre_mgrdirectsolversetup, HYPRE_MGRDIRECTSOLVERSETUP)
+( hypre_F90_Obj *solver,
+  hypre_F90_Obj *A,
+  hypre_F90_Obj *b,
+  hypre_F90_Obj *x,
+  hypre_F90_Int *ierr)
+{
+   *ierr = (hypre_F90_Int)
+           ( HYPRE_MGRDirectSolverSetup(
+                hypre_F90_PassObj (HYPRE_Solver, solver),
+                hypre_F90_PassObj (HYPRE_ParCSRMatrix, A),
+                hypre_F90_PassObj (HYPRE_ParVector, b),
+                hypre_F90_PassObj (HYPRE_ParVector, x) ) );
+}
+
+/*--------------------------------------------------------------------------
+ * HYPRE_MGRDirectSolverSolve
+ *--------------------------------------------------------------------------*/
+
+void
+hypre_F90_IFACE(hypre_mgrdirectsolversolve, HYPRE_MGRDIRECTSOLVERSOLVE)
+( hypre_F90_Obj *solver,
+  hypre_F90_Obj *A,
+  hypre_F90_Obj *b,
+  hypre_F90_Obj *x,
+  hypre_F90_Int *ierr )
+{
+   *ierr = (hypre_F90_Int)
+           ( HYPRE_MGRDirectSolverSolver(
```

Comment:
Should this be HYPRE_MGRDirectSolverSolve instead?

Reply:
Thanks for catching it! :)